core/src/main/kotlin/ch/ergon/dope/QueryBuilder.kt

```diff
@@ -22,7 +23,13 @@ class QueryBuilder {
 
     fun selectFrom(fromable: Fromable) = SelectClause(AsteriskExpression()).from(fromable)
 
+    fun selectFrom(bucket: AliasedBucket) = SelectClause(AsteriskExpression()).from(bucket)
```

Comment:
Why do we need this method anyway? An AliasedBucket is a fromable as well

Reply:
Yes, but we need to know, that it is an AliasedBucket, to add the `AS alias` to it